GameBoard/appsettings.json

```diff
@@ -1,11 +1,16 @@
 {
-  "ConnectionStrings": {  
-    "DefaultConnection": "Host=localhost;Port=5432;Database=gameboard_db;UserId=postgres;Password=postgres"
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=GameboardDB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
+    "GameboardStaging": "Server=tcp:gameboardio.database.windows.net,1433;Initial Catalog=Gameboard_Dev;Persist Security Info=False;User ID=gameboard_admin;Password=mbmgjkwmmsIO2019;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;",
```

Comment:
You've pushed our dev database password to a public repo. You'll need to change it and hide it from the public using User Secrets. You can ask @Johny7013 about User Secrets, since he was researching that lately, or read about it on the [MSDN](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-2.2&tabs=windows). To share the password with us you should probably use our GMail and either save it somewhere on the Google Drive or send it to us.

Reply:
Done.